var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -87,14 +87,22 @@ class Trilinos(CMakePackage):
             description='Enables the build of shared libraries')
     variant('debug',        default=False,
             description='Builds a debug version of the libraries')
-    variant('boost',        default=True, description='Compile with Boost')
-    variant('tpetra',       default=True, description='Compile with Tpetra')
-    variant('exodus', default=False, description='Compile with Exodus from SEACAS')
+    variant('boost',        default=True,  description='Compile with Boost')
+    variant('tpetra',       default=True,  description='Compile with Tpetra')
+    variant('exodus',       default=False, description='Compile with Exodus from SEACAS')
+
+    variant('dtk',          default=False, description='Enable DataTransferKit')
+    resource(name='dtk',
+             git='https://github.com/ornl-cees/DataTransferKit',
```

Comment:
Does it mean if I provide the hash it's going to behave like a release? How do I do that?
And still, is it possible to provide multiple versions for a resource?